react-file-uploads/client/src/UploadForm.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-unused-vars -- Remove me */
+const url = (path) => `${process.env.REACT_APP_BASE_URL}${path}`;
+
+export default  function UploadForm() {
+  async function handleSubmit(event) {
+
+    /* Prevent the browser's default behavior for form submissions.
+     * Create a `new` FormData object from the `event`.
+     *
+     * Use fetch() to send a POST request to http://localhost:8080/api/uploads.
+     * The body should be the form data object you created (not a JSON string).
+     * Headers are not necessary as fetch will use the correct Content-Type
+     * automatically (multipart/form-data).
+     *
+     * Parse the JSON response body and log the parsed response body.
+     * Log any errors to the console (using `console.error`).
+     *
+     * References:
+     * https://developer.mozilla.org/en-US/docs/Web/API/FormData/FormData
+     * https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#uploading_a_file
+     */
+    event.preventDefault();
+    const formData = new FormData(event.target);
+
+    const res = fetch('http://localhost:8080/api/uploads', {
+      method: "POST",
+      body: formData,
+  });
+    const jsonData = await res;
+
+    jsonData === undefined ? console.error('Something went wrong') : console.log(jsonData);
```

Comment:
You didn't check the response `ok` field. You don't catch any exceptions that are thrown.